app/views/articles/_sidebar.html.erb

```diff
@@ -8,16 +8,42 @@
     <% cache("display-area-left-#{rand(5)}-#{user_signed_in?}", expires_in: 5.minutes) do %>
       <% @left_sidebar_ad = DisplayAd.for_display("sidebar_left", user_signed_in?) %>
       <% if @left_sidebar_ad %>
-        <div class="crayons-card crayons-card--secondary p-3 crayons-sponsorship-widget" data-display-unit data-id="<%= @left_sidebar_ad.id %>">
-          <%= @left_sidebar_ad.processed_html.html_safe %>
+        <div class="crayons-card crayons-card--secondary crayons-sponsorship-widget" data-display-unit data-id="<%= @left_sidebar_ad.id %>">
+          <div class="crayons-sponsorship__header relative">
+            <div class="crayons-sponsorship__title"><%= Settings::Community.community_name %></div>
```

Comment:
Showing this name too but this is not part of this issue, so might remove it later.